director/__init__.py

```diff
@@ -101,6 +104,27 @@ def create_app(
     return app
 
 
+def build_celery_schedule(data):
+    # int or crontab
+    try:
+        schedule = float(data)
+    except ValueError:
+        try:
+            m, h, dw, dm, my = data.split(" ")
+
+            schedule = crontab(
+                minute=m,
+                hour=h,
+                day_of_week=dw,
+                day_of_month=dm,
+                month_of_year=my,
+            )
+        except Exception as e:
+            raise WorkflowSyntaxError()
```

Comment:
Can you forward the name of the related workflow please ? I think it will be useful to find the workflow with the bad syntax.

Reply:
Good point👌
Done